kubetest/e2e.go

```diff
@@ -506,16 +518,21 @@ func KubemarkTest(dump string, o options) error {
 	})
 
 	// Start new run
-	if popN, err := pushEnv("NUM_NODES", o.kubemarkNodes); err != nil {
+	if popN, err := pushEnv("NUM_NODES", nodes); err != nil {
 		return err
 	} else {
 		defer popN()
 	}
-	if popM, err := pushEnv("MASTER_SIZE", o.kubemarkMasterSize); err != nil {
+	if popM, err := pushEnv("MASTER_SIZE", masterSize); err != nil {
```

Comment:
Personally I would much rather explicitly set the masterSize in the job definition, rather than rely on a bash function (I guess we could also move the function into kubetest, but it just seems less flexible than --kubemark-master-size)

But will happily review a PR from you that removes it if that's the direction you want to go.